fedimint-api/src/task.rs

```diff
@@ -8,6 +8,8 @@ use std::sync::atomic::Ordering::SeqCst;
 use std::sync::Arc;
 use std::time::{Duration, Instant};
 
+#[cfg(target_family = "wasm")]
+use futures::channel::oneshot;
```

Comment:
you can use `tokio::sync` types on wasm now, btw

Reply:
Oh, good to know. We don't even seem to include `tokio` in wasm builds of `fedimint-api`. If anything changed recently that makes things easier, we should probably do a whole separate PR with a pass over all these `cfg`s and try to unify them.